### PR TITLE
Add PackagePaths as Parameter of ExecutionSession

### PR DESCRIPTION
**Pull request by mjkkirschner**
### Purpose

Try to address https://github.com/DynamoDS/DynamoWishlist/issues/71
* add parameter key for packagePaths
* add parameter
* add test
* add python test asserting we can reference and use this parameter.

Screen Shot 2020-07-01 at 1 20 38 PM

### Declarations

Check these if you believe they are true

- [x] The codebase is in a better state after this PR
- [x] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [x] The level of testing this PR includes is appropriate
- [x] User facing strings, if any, are extracted into `*.resx` files
- [x] All tests pass using the self-service CI.
- [x] Snapshot of UI changes, if any.
- [x] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.

### CODE REVIEW

**Comment by mjkkirschner** [comment]
all tests pass on self serve, but parallel has a concurrency issue with file load - will take a look.

**Comment by mjkkirschner** [comment]
I ran the parallel self serve again and all passed - so I think this is good to review now:
https://master-5.jenkins.autodesk.com/job/Dynamo/job/DynamoSelfServe/job/pullRequestValidation/837

File: src/NodeServices/ExecutionSession.cs [hunk 1]
```diff
@@ -69,5 +69,11 @@ public class ParameterKeys
         /// The duration of an execution covered by an <see cref="IExecutionSession"/>
```
Comment (aparajit-pratap) [inline src/NodeServices/ExecutionSession.cs:69]: I think we should make this class static as well.

Comment (mjkkirschner) [reply src/NodeServices/ExecutionSession.cs:69]: I agree, but I think that would sadly break the API. Do you prefer I obsolete this class, create a static version and use that one?

Comment (mjkkirschner) [reply src/NodeServices/ExecutionSession.cs:69]: or just risk the change?

Comment (aparajit-pratap) [reply src/NodeServices/ExecutionSession.cs:69]: Can we add a TODO to add it later?

Comment (mjkkirschner) [reply src/NodeServices/ExecutionSession.cs:69]: yep, will do.

Comment (mjkkirschner) [reply src/NodeServices/ExecutionSession.cs:69]: added and task created.

**Pull request merged**